Makefile

```diff
@@ -0,0 +1,5 @@
+all: compile
+
+compile:
```

Comment:
I'm no big fan of the typescript/javascript build, but this makefile is not complete.  IMO, this is more confusing then helping.